packages/components/src/actions.ts

```diff
@@ -228,11 +229,15 @@ export function toggleGridRowSelection(model: QueryGridModel, row: Map<string, a
 
             const selectedIds = checked ? model.selectedIds.push(stringKey) : model.selectedIds.delete(model.selectedIds.findIndex(item => item === stringKey));
 
-            updateQueryGridModel(model, {
+            const updatedModel = updateQueryGridModel(model, {
                 selectedState: selectedState,
                 selectedQuantity: selectedIds.size,
                 selectedIds: selectedIds
             });
+
+            if (Utils.isFunction(onSelectionChange)) {
```

Comment:
Is there a case when `onSelectionChange` will be defined but not a function?  This is a safer check, of course, so just curious.

Reply:
No, because of our use of TypeScript I don't think onSelectionChange could be defined without it being a function.